doc/source/getting_started/installation.rst

```diff
@@ -3,20 +3,61 @@
 ============
 Installation
 ============
+Xinference can be installed with ``pip`` on Linux, Windows, and MacOS. To run models using Xinference, you will need to install the backend corresponding to the type of model you intend to serve.
 
+If you aim to serve all supported models, you can install all the necessary dependencies with a single command::
 
-Installing from PyPI
-====================
+   pip install "xinference[all]"
```

Comment:
Using 'all' to install GGML dependencies might not always work. It's recommended to add a clear prompt here, guiding Mac (or the other) users to directly follow the steps below instead of using 'all' to install."

Reply:
Done.